solr/test-framework/src/java/org/apache/solr/cloud/SolrCloudTestCase.java

```diff
@@ -420,4 +425,68 @@ public static void ensureRunningJettys(int nodeCount, int timeoutSeconds) throws
     cluster.waitForAllNodes(timeoutSeconds);
   }
 
+
+  static final List<String> AUTH_METRICS_KEYS = Arrays.asList("errors", "timeouts", "requests", "authenticated", 
```

Comment:
I think it would be better to put this whole section in a subclass of SolrCloudTestCase, eg. SolrCloudAuthTestCase - there are only two tests that need this stuff but there are hundreds of tests that extend this base class and don't need this.

Reply:
Yep, a separate test base class for auth tests makes sense. Good idea. We could then push down some other utility methods from the tests as well, I see some duplication a few places.